index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```

Comment:
- [ ] [OPTIONAL] I suggest that you link the CSS to the HTML file. This will help us to have the styling of `h1` have an effect on the heading.